enhancements/monitoring/alerts-ui-management.md

```diff
@@ -0,0 +1,337 @@
+---
+title: alerts-ui-managment
+authors:
+  - "@sradco"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2025-07-24
+last-updated: 2025-07-30
+tracking-link:
+  - "https://issues.redhat.com/browse/CNV-46597"
+  - "https://issues.redhat.com/browse/CNV-62972"
+---
+# New APIs to support Alerts UI Management in OpenShift
+
```

Comment:
Can we link to existing enhancement proposals which are relevant to this effort? Namely https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alert-overrides.md and https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/user-workload-monitoring.md.

In particular the first one calls out UI/UX issues with the console.